aeneas/src/main/CLOptions.v3

```diff
@@ -149,8 +149,12 @@ component CLOptions {
 		"Normalize ranges using RangeStart's to support off-heap ranges.");
 	def SET_EXEC		= compileOpt.newBoolOption("set-exec", true,
 		"Automatically set execute permission for compiled binaries.");
-	def USE_GLOBALREGALLOC = compileOpt.newMatcherOption("global-regalloc",
+	def USE_GLOBALREGALLOC	= compileOpt.newMatcherOption("global-regalloc",
 		"Enable global register allocator.");
+	def UNBOX_VARIANTS	= compileOpt.newBoolOption("unbox-variants", false,
+		"Enable variant unboxing features.");
+	def UNBOX_ALL		= compileOpt.newBoolOption("unbox-all", false,
```

Comment:
What about `-unbox-variants=<filter>` that accepts a glob, with the default empty pattern being "all"?

Also, about what `-unbox-variant-cases=<list>` that takes a list of integers and marks all corresponding cases as `#unboxed`? That would give a fuzzer or a manual tester a lot of control to try out many combinations.

Reply:
I'll put the implementation of the `-unbox-variant-cases=<list>` in the PR with case-specific unboxing, but I've changed the appropriate variable here.